### ISSUE TITLE
(URGENT) Unable to fetch `JCGM` documents

### ISSUE THREAD

**Issue by ronaldtse** [relaton/relaton-bipm#56]
```
[relaton-bipm] (BIPM JCGM 100:2008) Fetching from Relaton repository ...
[relaton-bipm] WARNING: Incorrect reference: `JCGM 100:2008`
[relaton-bipm] (BIPM JCGM 101:2008) Fetching from Relaton repository ...
[relaton-bipm] WARNING: Incorrect reference: `JCGM 101:2008`
[relaton-bipm] (BIPM JCGM 200:2008) Fetching from Relaton repository ...
[relaton-bipm] WARNING: Incorrect reference: `JCGM 200:2008`
Bibliography: Could not retrieve BIPM JCGM 100:2008: no access to online site
Bibliography: Could not retrieve BIPM JCGM 101:2008: no access to online site
Bibliography: Could not retrieve BIPM JCGM 200:2008: no access to online site
```

**Comment by andrew2net** [comment]
@ronaldtse shouldn't we use this [reference structures](https://github.com/relaton/relaton-bipm#reference-structure-for-metrologia-documents)?

**Comment by ronaldtse** [comment]
@andrew2net no, these are separate documents.

**Comment by andrew2net** [comment]
Implemented in v1.16.6
```sh
$ relaton fetch --no-cache 'JCGM 100:2008'
[relaton-bipm] (JCGM 100:2008) Fetching from Relaton repository ...
[relaton-bipm] Downloaded index from https://raw.githubusercontent.com/relaton/relaton-data-bipm/master/index2.zip
[relaton-bipm] (JCGM 100:2008) Found: `JCGM 100:2008`
<bibdata type="standard" schema-version="v1.2.4">
...

$ relaton fetch --no-cache 'JCGM 101:2008'
[relaton-bipm] (JCGM 101:2008) Fetching from Relaton repository ...
[relaton-bipm] (JCGM 101:2008) Found: `JCGM 101:2008`
<bibdata type="standard" schema-version="v1.2.4">
...

$ relaton fetch --no-cache 'JCGM 200:2008'
[relaton-bipm] (JCGM 200:2008) Fetching from Relaton repository ...
[relaton-bipm] (JCGM 200:2008) Found: `JCGM 200:2008`
<bibdata type="standard" schema-version="v1.2.4">
...
```
